packages/tokens/src/global/borders.ts

```diff
@@ -0,0 +1,11 @@
+export interface BorderProps extends Array<number> {
+  zero?: number;
+  small?: number;
+  medium?: number;
+}
+
+const border: BorderProps = [0, 1, 2];
+
+[border.zero, border.small, border.medium] = border;
```

Comment:
**suggestion**: I think we can remove this line. I'm not really following what it's doing 🤔